larky/src/test/resources/test_fco_operation.star

```diff
@@ -1,191 +1,77 @@
+load('urllib/request', 'Request')
+load("testlib/asserts", "asserts")
 
 
+def _name():
+    return "github.com/verygoodsecurity/xxxx/e75fcdb7-7b86-4384-870e-a24fdca31ef5/tntzr6gpm1s/16c6e5c7-8a2a-4e6e-9e78-2ffc11650a52/38c28e87-3927-4931-9be6-6c699e1954e7/Config"
 
-# - name: github.com/verygoodsecurity/common/http/header/Get
-#       config:
-#         header: "Private-Key"
-#         var: "ctx.Private-Key"
-#
-
-
-# from vgs_messages.http_pb2 import HttpMessage
-# from vgs_messages.http_pb2 import HttpHeader
-# from vgs_messages.http_pb2 import HttpPhase
-
-load('urllib/request', 'Request')
+# Sort the json object alphabetically
+def _get_second_item(pair):
+    return pair[1]
 
-def name():
-    return "github.com/verygoodsecurity/xxxx/e75fcdb7-7b86-4384-870e-a24fdca31ef5/tntzr6gpm1s/16c6e5c7-8a2a-4e6e-9e78-2ffc11650a52/38c28e87-3927-4931-9be6-6c699e1954e7/Config"
+def operate(http_message):
```

Comment:
```suggestion
def process(http_message, ctx):
```

the js implementation is `VGS.process(payload, ctx)` so keeping the function signature aligned will make it easier for people to grok. ctx is a dict and payload is polymorphic.